go.mod

```diff
@@ -99,12 +98,18 @@ require (
 	modernc.org/mathutil v1.0.0
 )
 
+require (
```

Comment:
why the additional require block?

Reply:
I was curious about that as well, I ran a `go mod tidy` (with go v17) and that's what it came up with.

Reply:
https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17